src/annotator/sidebar.js

```diff
@@ -19,12 +21,6 @@ import { ToolbarController } from './toolbar';
 // Minimum width to which the frame can be resized.
 const MIN_RESIZE = 280;
 
-const defaultConfig = {
-  BucketBar: {
-    container: '.annotator-frame',
```

Comment:
The `container` option is no longer set anywhere but `BucketBar` still has code to handle it. Can you open a follow-up PR to remove that code?

Reply:
Yes, I will. In addition, we may want to remove anything that make it work as a 'plugin'.

Reply:
https://github.com/hypothesis/client/issues/2907